tracer/build/_build/Build.Steps.cs

```diff
@@ -893,6 +893,27 @@ async Task DownloadWafVersion(string libddwafVersion = null, string uncompressFo
                 .EnableNoDependencies()
                 .SetTargets("BuildDependencyLibs")
             );
+
+            // The live debugger dependency lib _sometimes_ has to be x86/x64, and _sometimes_ has to be AnyCPU apparently
```

Comment:
I'm honestly not clear on this, in terms of why would it have to sometimes be AnyCPU (should be just x86/x64). Having said that, I'm fine with merging this as is, since @shurivich has been planning to work on this in the near term and refactor the way this dependency works, so if this works, great - we can just leave it as-is.

Reply:
Something has changed in the way the .NET SDK uses the target platform supplied in build/test. One of the other changes I had to make is _removing_ the `SetTagetFramework()" from the integration tests, and running those as "AnyCPU", though we still run against x64/x86, and the samples are built against x64/x86. The whole thing is a bit of a mess tbh, not just the debugger side 😬 I'm trying to do some cleanup in a separate PR but it's tough...